pkg/cmd/backuprestore/restoreutils.go

```diff
@@ -106,3 +119,32 @@ func restore(configDir, dataDir, backupDir string) error {
 	return nil
 
 }
+
+func waitForStaticPodsToStop() error {
+	runtimeClient, runtimeConn, err := getRuntimeCrioClient()
+	if err != nil {
+		return err
+	}
+	defer closeConnection(runtimeConn)
```

Comment:
1. This function is tiny and only used in one place, consider just inlining it as a closure
2. The error needs handled (logging)